roles/manage-gcp-infra/tasks/main.yaml

```diff
@@ -0,0 +1,16 @@
+---
+
+- name: upload image if necessary
```

Comment:
What is this for? Can it be removed?

Reply:
I had plan for it but will delete for now.